### PR TITLE
Completed prep-css-box-modele exercise.

### PR DESCRIPTION
**Pull request by kelvin-lao-01**
Screen Shot 2022-08-03 at 10 55 58 AM
Screen Shot 2022-08-03 at 10 56 03 AM

### CODE REVIEW

File: prep-css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-width: 3px;
+  p.double { border-style: double; }
```
Comment (thebearingedge) [inline prep-css-box-model/styles.css:11]: Hi Kelvin, you should move this rule set outside of the rule set for the button, it's not applying the double border style unfortunately.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-box-model/styles.css (hunk 1 above)
Comment (kelvin-lao-01) [reply prep-css-box-model/styles.css:11]: gotcha! will do!

**Comment by kelvin-lao-01** [comment]
Screen Shot 2022-08-03 at 10 55 58 AM
Screen Shot 2022-08-03 at 10 56 03 AM

**Comment by kelvin-lao-01** [comment]
Screen Shot 2022-08-03 at 10 50 23 PM
Screen Shot 2022-08-03 at 10 56 03 AM

File: prep-css-box-model/styles.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-width: 3px;
+  border-color: darkgray;
+}
+p.double { border-style: double; }
```
Comment (thebearingedge) [inline prep-css-box-model/styles.css:13]: Ok, better, but you want to make your buttons have a double border

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by kelvin-lao-01** [comment]
Screen Shot 2022-08-04 at 9 47 23 AM
Screen Shot 2022-08-04 at 9 47 37 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request closed without merging**